### PR TITLE
Adaptable Ingress Guide

### PR DESCRIPTION
**Pull request by marians**
See https://github.com/giantswarm/giantswarm/issues/1514

To be tested via https://github.com/giantswarm/docs/pull/210

### CODE REVIEW

File: content/guides/accessing-services-from-the-outside/index.md [hunk 1]
```diff
@@ -11,37 +11,96 @@ tags = ["tutorial"]
...
 ## Setting up a Public Ingress {#public-ingress}
 
-Your Giant Swarm cluster comes with an Ingress Controller based on [NGINX](https://github.com/kubernetes/ingress/blob/master/controllers/nginx/), which we run for you in your cluster. You can expose Services publicly by setting up a simple Ingress. You can do this with an ingress manifest (e.g. `myingress.yaml`) that looks like following template (replace <fields> accordingly) in the same namespace as the service you want to expose. 
+Before we explain how to set up Ingress for a Service, please read the next section carefully to make sure the guide matches your situation as a Giant Swarm user.
+
+### Knowing your Ingress Base Domain {#base-zone}
+
+Setting up Ingress means to make Services publicly available via DNS names. For an application facing the public, you will eventually want to set up names ending in your own domain.
+
+However for development or test purposes, you can use the fact that every Giant Swarm installation maps to a DNS zone. We call this the **Ingress Base Domain**.
```
Comment (puja108) [inline content/guides/accessing-services-from-the-outside/index.md]: `However` -> `However,`

File: content/guides/accessing-services-from-the-outside/index.md [hunk 2]
```diff
@@ -11,37 +11,96 @@ tags = ["tutorial"]
...
+
+- `INGRESS_NAME`: This name identifies the ingress within your namespace. As with all other kubernetes, it's up to you how you name it.
+- `NAMESPACE`: The namespace of the service you'd like to expose. The ingress must reside in the same namespace, so it's useful to make it part of the metadata in the manifest.
+- `PREFIX`: A subdomain name of your choice. This must be unique among all ingresses of this cluster. If you like, you can make this the same as `INGRESS_NAME`.
+- `CLUSTER_ID`: The ID of the Kubernetes cluster the service you'd like to expose is running on. This should be a string consisting of five letters and numbers.
+- `SERVICE_NAME`: The name of your service, as defined in the `metadata.name` attribute of your service's manifest.
+- `SERVICE_PORT`: The port of your service to connect to, as defined in the `spec.ports` array of your service's manifest.
+
+The `host` attribute defines the fully qualified host name via which your service will be accessible. As you can see, it consists of your ingress name, your namespace, your cluster ID, and finally the ingress base zone.
```
Comment (puja108) [inline content/guides/accessing-services-from-the-outside/index.md]: I guess you forgot to change this ^ should be prefix, cluster ID, base zone

File: content/guides/accessing-services-from-the-outside/index.md [hunk 3]
```diff
@@ -11,37 +11,96 @@ tags = ["tutorial"]
...
 
-You can apply the ingress resource with `kubectl apply -f myingress.yaml`. You can also leave out the `namespace` line in the yaml and instead use `kubectl apply --namespace <namespace> --filename myingress.yaml`.
+It will take a moment for the Ingress to be created. You can look up the new Ingress using `kubectl` like this (replacing `NAMESPACE` and `INGRESS_NAME`):
+
+```nohighlight
+kubectl -n NAMESPACE describe ing/INGRESS_NAME
+```
 
-A few moment later you will be able to access your service publicly at:
+Once the Ingress is up, you will be able to access your service publicly at this URL like this (again replacing the placeholders):
```
Comment (puja108) [inline content/guides/accessing-services-from-the-outside/index.md]: `this URL` -> `a URL`

File: content/guides/accessing-services-from-the-outside/index.md (hunk 2 above)
Comment (marians) [inline content/guides/accessing-services-from-the-outside/index.md]: right...

**Comment by marians** [comment]
@puja108 Changes are done

**Pull request merged**